sasmodels/conversion_table.py

```diff
@@ -255,7 +255,7 @@
                 "mphi:sld_solvent": "M_phi_solv",
                 "up:frac_i": "Up_frac_i",
                 "up:frac_f": "Up_frac_f",
-                "up:angle": "Up_theta"
+                "up:angle": "up_angle"
```

Comment:
I notice that `Up_frac_i` and `Up_frac_f` rather than `up_frac_i` and `up_frac_f`. The rest of the code is using lower case, so these should be changed as well.

Reply:
Magnetic parameters are fixed and changed consistently to lower case.